platform/ios/src/MGLMapView.h

```diff
@@ -226,6 +226,15 @@ IB_DESIGNABLE
 - (IBAction)reloadStyle:(id)sender;
 
 /**
+ A control indicating the scale of the map.
+ 
+ The scale bar becomes visible when a user interacts with the
+ map and fades out when the interaction stops. It’s positioned
+ in the upper-left corner.
+ */
+@property (nonatomic, readonly) UIView *scaleBarView;
```

Comment:
“Scale bar view” is redundant. Let’s call it either a “scale bar” (`scaleBar`, `MGLScaleBar`) or a “scale view” (`scaleView`, `MGLScaleView`).